composer.json

```diff
@@ -12,11 +12,20 @@
     "require": {
     },
     "require-dev": {
-        "thefrozenfire/swagger-client": "^2.0"
+        "thefrozenfire/swagger-client": "^2.0",
+        "phpunit/phpunit": "~4.8.28"
     },
     "autoload": {
         "psr-0": {
             "CallFire": "src/"
-        }
+        },
+        "classmap": [
+            "test/"
+        ]
+    },
+    "scripts": {
+        "post-install-cmd": [
+            "php codeHintsInstall.php"
```

Comment:
Could we encapsulate the codeHintsInstall.php functionality in a class instead, and use a class method reference? [See here](https://getcomposer.org/doc/articles/scripts.md#defining-scripts). This will prevent any sort of issues with the working path not resolving (e.g. if composer install is run outside of the project directory).